doc/developers-guide/developers-guide.md

```diff
@@ -62,8 +62,8 @@ The entry point is `flake.nix` in the root of the project, where the inputs and
 
 `nix develop .#rust` will create a shell env for developing rust.
 
-`nix build .?submodules=1` will build the default package (cln).
+`nix build` will build the default package (cln).
 
-`nix flake check .?submodules=1` will build the cln and rust packages. Rust tests are run during the build. There are also checks to run cargo audit and nixfmt.
+`nix flake check` will build the cln and rust packages. Rust tests are run during the build. There are also checks to run cargo audit and nixfmt.
 
-If you have nix installed you can use `nix run "git+https://github.com/hashrelay/lightning?ref=flake&submodules=1#lightningd"` to run lightningd without having to manually clone the repo. This make use of the flake output apps.
+If you have nix installed you can use `nix run "git+https://github.com/hashrelay/lightning?ref=flake#lightningd"` to run lightningd without having to manually clone the repo. This makes use of the flake output `apps`.
```

Comment:
While this has existed before your PR, I fail to see why we should refer to a fork. It seems like poor practice to direct people to an external source that we don't control.
Could we use 
```
nix run "git+https://github.com/ElementsProject/lightning?ref=<release-branch>#lightningd"
```
instead?

If so we can change this in a different PR.

Reply:
Yes, that's a mistake, it should have said ElementsProject from the start. I've fixed it now.